cfgov/v1/auth_forms.py

```diff
@@ -36,14 +36,6 @@ class UserEditValidationMixin(PasswordValidationMixin):
     user_attribute = 'instance'
 
 
-class CFGOVUserCreationForm(UserEditValidationMixin, UserCreationForm):
-    pass
```

Comment:
Yeah, @richaagarwal you're right we haven't been enforcing that. I think fixing it is out of scope for this PR.